test/http/api.go

```diff
@@ -0,0 +1,27 @@
+package http
```

Comment:
license + some comments on the methods

Reply:
I don't really care about comments on test funcs?

Reply:
I'm still slow at reading code / processing new concepts. Oneliners telling me what to expect or when to use it help a lot. Having to go to a test function to figure out what it does requires moving through code and distracts from what you are trying to test.

Although these functions are probably simple enough to understand